SharpMUSH.Client/Pages/Admin/Layout/LayoutEditor.razor.css

```
@@ -0,0 +1,104 @@
+
+.le-btn {
+    display: inline-flex; align-items: center; gap: 0.375rem; padding: 0.4375rem 0.8125rem; border-radius: var(--radius);
+    border: 1px solid var(--border); background: var(--surface-2); color: var(--text);
+    font-family: var(--font-ui); font-size: 0.78125rem; font-weight: 500; cursor: pointer; transition: all 0.12s;
+}
+.le-btn:hover:not(:disabled) { border-color: rgba(var(--glow), 0.4); }
+.le-btn:disabled { opacity: 0.5; cursor: default; }
+.le-btn--primary { background: var(--accent); color: var(--accent-on); border-color: transparent; font-weight: 600; }
+.le-btn--primary:hover:not(:disabled) { filter: brightness(1.08); }
+
+.layedit ::deep .le-board { display: grid; grid-template-columns: 260px minmax(0, 1fr); gap: 1rem; }
```

Stylelint (17.13.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)